packages/sdk/src/across/README.md

```diff
@@ -0,0 +1,32 @@
+# Across Utilities
+
+Across is a system that uses UMA contracts to quickly move tokens across chains. This contains various utilities to support applications
+on across.
+
+## Fee Calculator
+
+Calculates lp fee percentages when doing a transfer.
+
+### Usage
+
+See tests for more documentation: [Fee Calculator Test]("./feeCalculator.test.ts")
+
+```ts
+import * as uma from "@uma/sdk"
+
+const { calculateApyFromUtilization, calculateRealizedLpFeePct } = uma.across.feeCalculator
+
+// sample interest rate model. note these tests are in JS and so we can impose the RateModel type.
```

Comment:
what do you mean `these tests are in JS and so we can impose the RateModel type.`? do you mean the code snippet here?

Reply:
yea i copied and pasted the comment from test. will update